src/GraphQL.Analyzers.Tests/FieldBuilderAnalyzerTests.cs

```diff
@@ -454,6 +454,55 @@ public MyGraphType()
         await VerifyCS.VerifyCodeFixAsync(source, expected, fix).ConfigureAwait(false);
     }
 
+    [Fact]
+    public async Task EmptyLineWithWhiteSpacesBeforeField_ArgumentsFormattingPreservedAsync()
+    {
+        const string source = """
+            using GraphQL.Types;
+
+            namespace Sample.Server;
+
+            public class MyGraphType : ObjectGraphType
+            {
+                public MyGraphType()
+                {
+                    // next line contains whitespaces
+                    
+                    Field<StringGraphType>("name", "description", null,
+                        resolve: context => Resolve(
+                            "text"
+                        ));
+                }
+
+                public string Resolve(string s1) => s1;
+            }
+            """;
+
+        const string fix = """
+            using GraphQL.Types;
+
+            namespace Sample.Server;
+
+            public class MyGraphType : ObjectGraphType
+            {
+                public MyGraphType()
+                {
+                    // next line contains whitespaces
+                    
+                    Field<StringGraphType>("name").Description("description")
+                        .Resolve(context => Resolve(
+                            "text"
+                        ));
+                }
+
+                public string Resolve(string s1) => s1;
+            }
+            """;
+
+        var expected = VerifyCS.Diagnostic(FieldBuilderAnalyzer.DoNotUseObsoleteFieldMethods).WithSpan(11, 9, 14, 15);
```

Comment:
```suggestion
        // ensure whitespace was not removed from input sample
        source.Replace("\r", "").ShouldContain("// next line contains whitespaces\n    ");

        var expected = VerifyCS.Diagnostic(FieldBuilderAnalyzer.DoNotUseObsoleteFieldMethods).WithSpan(11, 9, 14, 15);
```

Reply:
I'm thinking that it is likely enough that some other formatting tool in the future may remove this whitespace, and the test should still pass.  So with a couple extra lines of code we can ensure the test is correct.

Reply:
Makes sense.